sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryUtils.java

```diff
@@ -269,6 +269,12 @@ public static Schema fromTableSchema(TableSchema tableSchema) {
     return fromTableFieldSchema(tableSchema.getFields());
   }
 
+  /** Convert a list of BigQuery {@link TableFieldSchema} to Avro {@link org.apache.avro.Schema}. */
+  public static org.apache.avro.Schema toGenericAvroSchema(
```

Comment:
Why adding an extra call?

Reply:
This call works as proxy to `BigQueryAvroUtils`, which is not a public class, so it's not accessible from other packages. I did not want to make the whole class public, so instead I added this call to `BigQueryUtils`, which is a public class with similar calls for schema conversions.